nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
```

Comment:
BitHistory of c++ and python implemented in different fashion. Either python classifier adapts itself to re-use BitHistory schema from c++ (or vice-versa) or than change python bit histories to have the same architecture of its c++ version.